### DISCUSSION TITLE
AnimationMixer, AnimationPlayer, animation_finished(anim_name: StringName) => animation_finished(anim_name, backwards:bool)

### DISCUSSION THREAD

**Discussion by AlexeyPe** [godotengine/godot-proposals#10109] [Engine Core]
We have .play(), we also have .play_backwards()
Why don't we have backwards in the signal animation_finished?
I can define backwards using current_animation_position, but then why is there a play_backwards?...

**Comment by Calinou** [comment]
I'm not sure if this would be welcome, given this comment: https://github.com/godotengine/godot/pull/89525#issuecomment-2099586647

My PR that adds `animation_looped` signal adds a `backwards` parameter to that signal, but I didn't touch `animation_finished`. Also, I believe adding a new optional argument to a signal breaks compatibility, because signal receiver functions can't be called if too many arguments are passed by the emitter.
